libs/sdk-ui-tests/scenarios/charts/_infra/responsiveScenarios.ts

```diff
@@ -13,20 +13,31 @@ export function responsiveScenarios<T extends VisProps>(
     component: React.ComponentType<T>,
     baseProps: UnboundVisProps<T>,
     sizes: Array<IResponsiveSize>,
+    generateInsight: boolean = false,
```

Comment:
Minor detail: This will trigger a sonarqube smell about parameters with default value needing to be last. Can we avoid it? Maybe by providing default for the customizer?

Reply:
All code smells fixed